features/flaw_affects.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Flaw detail testing
+
+    Background: Before run scenarios
+      Given I am an analyst AND I am logged into OSIM
+      And I set the bugzilla api key and jira api key
+    
+    @update
+    Scenario: Update affects
+      When I update the affects of the flaw and click 'Save Changes' button
```

Comment:
Meet error, test on flaw https://osim-stage.prodsec.redhat.com/flaws/CVE-2018-82341 :
Feature: Flaw detail testing # features/flaw_affects.feature:1

  Background: Before run scenarios  # features/flaw_affects.feature:3

  @update
  Scenario: Update affects                                                # features/flaw_affects.feature:8
    Given I am an analyst AND I am logged into OSIM                       # features/steps/common_steps.py:13 7.861s
    And I set the bugzilla api key and jira api key                       # features/steps/common_steps.py:18 6.013s
    When I update the affects of the flaw and click 'Save Changes' button # features/steps/flaw_affects.py:8 2.545s
      Traceback (most recent call last):
        File "/Users/axuan/osim/venv/lib/python3.10/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/Users/axuan/osim/venv/lib/python3.10/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
        File "features/steps/flaw_affects.py", line 25, in step_impl
          ps_module = affects_page.get_an_available_ps_module(
        File "/Users/axuan/osim/features/pages/affects_page.py", line 55, in get_an_available_ps_module
          module_list.remove(f'{n}')
      ValueError: list.remove(x): x not in list

    Then All changes are saved                                            # None


Failing scenarios:
  features/flaw_affects.feature:8  Update affects

0 features passed, 1 failed, 0 skipped
0 scenarios passed, 1 failed, 0 skipped
2 steps passed, 1 failed, 1 skipped, 0 undefined
Took 0m16.419s

Reply:
Updated.